public/css/form.css

```
@@ -0,0 +1,24 @@
+.banner {
+    background-color: green;
+}
+form {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    text-align: center;
+}
+form table {
+    display: inline-block;
+    border-collapse: collapse;
+}
+ th, tr, td {
```

Mỗi 1 query selector hãy viết trên 1 dòng

```suggestion
 th,
 tr,
 td {
```